packages/wrangler/src/api/startDevWorker/LocalRuntimeController.ts

```diff
@@ -220,18 +122,16 @@ async function convertToConfigBundle(
 		initialPort: undefined,
 		initialIp: "127.0.0.1",
 		rules: [],
-		inspectorPort:
-			event.config.dev?.inspector?.port ??
-			(await getPort({ port: DEFAULT_INSPECTOR_PORT })),
+		inspectorPort: 0,
```

Comment:
does this mean we're forever ignoring whatever's been passed in `dev.inspector.port`?

Reply:
No, it's used as the port for the inspector proxy (this value is for the user worker inspector port). See https://github.com/cloudflare/workers-sdk/blob/7d93a60deeb8b64ca8ff8be66860ffbda4b860ad/packages/wrangler/src/dev.tsx#L773 in the existing flow, although I'm realising that this should probably include a `getPort` call as well